pipelines/_ingredients/ApPipeWithFakes.yaml

```diff
@@ -33,6 +33,10 @@ tasks:
       connections.output_catalog: fakes_initial_pvi_catalog
       connections.injection_prefix: parameters.injection_prefix
       connections.injected_prefix: parameters.injected_prefix
+  loadDiaCatalogs:
+    class: lsst.ap.association.LoadDiaCatalogsTask
+    config:
+      apdb_config_url: parameters.apdb_config
```

Comment:
Please add this to the `ApPipe` subset as well -- that subset is defined to include _everything_.